docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
A quick test suggests setting this to ```../``` can allow moving the docker-compose.yml into ```docker/```, might have other impacts.

not required but looks like would work.